etc/profile-a-l/atool.profile

```diff
@@ -13,7 +13,7 @@ include allow-perl.inc
 noroot
 
 # without login.defs atool complains and uses UID/GID 1000 by default
-private-etc alternatives,group,ld.so.cache,ld.so.preload,login.defs,passwd
+private-etc alternatives,group,ld.so.cache,ld.so.preload,login.defs,passwd,resolv.conf
```

Comment:
This profile includes archiver-common.inc, which has `net none`.

So what does this change fix?

Reply:
It's the same issue as #5608. Maybe it doesn't make much sense to have private-etc in an archiver tool in the first place?